sigstore-java/src/main/java/dev/sigstore/KeylessSigner.java

```diff
@@ -191,9 +191,11 @@ public KeylessSigner build()
       Preconditions.checkNotNull(sigstoreTufClient, "sigstoreTufClient");
       sigstoreTufClient.update();
       var trustedRoot = sigstoreTufClient.getSigstoreTrustedRoot();
-      var fulcioClient = FulcioClient.builder().setCertificateAuthority(trustedRoot).build();
+      var fulcioClient =
+          FulcioClient.builder().setUri(trustedRoot.getCAs().current().getUri()).build();
       var fulcioVerifier = FulcioVerifier.newFulcioVerifier(trustedRoot);
-      var rekorClient = RekorClient.builder().setTransparencyLog(trustedRoot).build();
+      var rekorClient =
+          RekorClient.builder().setUri(trustedRoot.getTLogs().current().getBaseUrl()).build();
```

Comment:
this bit will be modified in the future to be initalized from the signing init config.

Reply:
How does this handle if multiple logs or CAs are active? Does `current` take the first?

Reply:
Yeah that's just an artifact of the older code that was initializing under the assumption there was only one CA. Once https://github.com/sigstore/protobuf-specs/pull/277 messages are created, we'll init from that.